docs/3.0.0-beta.x/community/contribute-with-docs.md

```diff
@@ -0,0 +1,486 @@
+
+**Tutorials** and **How-tos** are sometimes by a members of the community. A byline appears towards the title with a **Written by: Paul Bocuse**. If you make _significant_ revisions to the **Tutorial** or **How-tos**, then the byline would change and include your name, like so: **Original Author: Paul Bocuse, Revised by: Your Name**. You should link to your GitHub from your name.
+
+**Examples** are generally added to **Reference Topics** and **How-tos**. The examples are added directly in the existing content.
+
+### Submitting an Externally Hosted Tutorial
+
+If you have written a **Tutorial**, **How-to** or other teaching material that is _outside_ of the documentation. Then you may submit a pull request with a link to your guide on the [Articles](/3.0.0-beta.x/articles/) page.
+
+### Docs Style Guide
+
+This style guide exists to show and explain the standards and style of how our documentation is written. We are offering this style guide to help you stay consistent with the existing documentation.
+
+---
+
+#### Favor _Yarn_ commands over _npm_ commands
+
+We recommend that users develop using the Yarn package manager. Therefore, the documentation shows commands first using `yarn` and then showing the equivalent `npm` command. Please consider the below examples:
+
+**Example 1**
+
+- Install Strapi with the following commands:
+
+Using **Yarn**:
+
+```
+yarn create strapi-app my-project --quickstart
+```
+
+Using **npm/npx**:
+
+```
+npx create-strapi-app my-project --quickstart
+```
+
+**Example 2**
+
+- You can access the Strapi Help instructions from your command line with:
+
+Using **Yarn**:
+
+```
+yarn strapi
+```
+
+Using **npm**:
+
+```
+npm run strapi help
+```
+
+---
+
+#### When giving instructions, say, _"you"_ not _"we"_
+
+Clear and concise instructions and language can help individuals to get the full advantages from using the Docs. Whenever an action needs doing, you use the word, "you" rather than "we." To clarify this, consider the following examples:
+
+**Example 1**
+
+1. **"You will next open your Chrome browser. Then, open a tab."**
+2. **"We will next open our Chrome browser. Then, we will open a tab."**
+
+In the above, the #1 phrase is precise, and the reader knows what is expected. In the #2 phrase, the additional words "we will" is needed to keep the sentence relatively clear. **Note:** "Then, we open a tab." - is awkward and unclear.
+
+**Example 2**
+
+1. **"Download Node.js and install it onto your computer."**
+2. **"We download Node.js, and then we install it onto our computers."**
+
+In the above, the #1 phrase is precise. #2 is actually confusing.
+
+Therefore, when you write instructions, use "you." This guide helps to keep the documentation clearly written.
+
+---
+
+#### Ensure to use trade, project and service provider names correctly
```

Comment:
to use trade as trademark?

Reply:
Yeah, upon review "trademark" is more clear.  Thanks.